spec/factories/organizations.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :organization do
+    name { "MyString" }
+    address { "MyString" }
+    phone { "" }
+    email { "MyString" }
+    welcomeText { "MyText" }
```

Comment:
La misma convención aplica acá:
```suggestion
    welcome_text { "MyText" }
```
Al correr nuevamente la migración midificada se debería arreglar automáticamente. Revisar por las dudas.

Reply:
Hecho